homeassistant/components/pjlink/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants for the PJLink integration."""
```

Comment:
Could the existing constants be moved in a preliminary PR?

Reply:
This has not been addressed. Please make this a preliminary PR.

Reply:
#91749